packages/chat/src/components/LinkPreview/styles.css.ts

```diff
@@ -0,0 +1,150 @@
+import { style } from '@vanilla-extract/css';
+import { recipe } from '@vanilla-extract/recipes';
+
+import { COLORS } from '@/styles/colors';
+import { FAMILY, lineClamp } from '@/styles/font';
+import { SIZES } from '@/styles/sizes';
+import { transition } from '@/styles/transitions';
+
+export const linkPreviewContainer = recipe({
+  base: [
+    {
+      display: 'flex',
+      borderRadius: SIZES.radius.sm,
+      padding: '6px 8px 6px 6px',
+      backgroundColor: COLORS.NEUTRAL_LIGHT[50],
+      fontFamily: FAMILY,
+      transition: transition(['color', 'background-color']),
+      height: 64,
+      ':hover': {
+        cursor: 'pointer',
+        backgroundColor: COLORS.NEUTRAL_LIGHT[100],
+      },
+      ':active': {
+        backgroundColor: COLORS.NEUTRAL_LIGHT[200],
+      },
+    },
+  ],
+
+  variants: {
+    inline: {
+      true: {
+        height: 43,
+        padding: '4px 8px 4px 4px',
+        backgroundColor: COLORS.NEUTRAL_LIGHT[100],
+        ':hover': {
+          backgroundColor: COLORS.NEUTRAL_LIGHT[200],
+        },
+        ':active': {
+          backgroundColor: COLORS.NEUTRAL_LIGHT[300],
+        },
+      },
+    },
+  },
+});
+
+export const linkPreviewImageContainer = recipe({
+  base: {
+    position: 'relative',
+    width: 52,
+    height: 52,
+    flexShrink: 0,
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderRadius: SIZES.radius.xxxs,
+    marginRight: 8,
+    backgroundColor: COLORS.NEUTRAL_DARK[9006],
+    overflow: 'hidden',
+  },
+
+  variants: {
+    inline: {
+      true: {
+        width: 35,
+        height: 35,
+      },
+    },
+    loading: {
+      true: {
+        boxShadow: 'none',
+      },
+    },
+  },
+});
+
+export const linkPreviewImage = style({
+  width: '100%',
+  '::after': {
+    content: '',
+    position: 'absolute',
+    height: '100%',
+    width: '100%',
+    top: 0,
+    right: 0,
+    display: 'block',
+    boxShadow: '0px -1px 0px 0px rgba(0, 0, 0, 0.04) inset',
```

Comment:
This is the only way I think you can add an 'inset' shadow over an image (as in Mike's design)